modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -351,7 +352,7 @@
                   expect(json_response.count).to eq(1)
                   claim = json_response['data'].first
                   expect(claim['attributes']['baseEndProductCode']).to eq('400')
-                  expect(claim['attributes']['status']).to eq('PREPARATION_FOR_NOTIFICATION')
+                  expect(claim['attributes']['status']).to eq('EVIDENCE_GATHERING_REVIEW_DECISION')
```

Comment:
adjust for [above](https://github.com/department-of-veterans-affairs/vets-api/pull/16011/files#r1544914926)